docs/Releases.md

```diff
@@ -13,6 +13,307 @@ use a date based versioning system.  Now, a release version can look
 like `17.11` where the "major" number is the year and the "minor" number
 is the month.
 
+## v19.05 (5/19): Shared Core Mode, Major Architectureal/API/Initialization/Signal Handling Changes, Advanced Rings Changes, Stats Updates, CI PR Review, LPM Firewall NF, Payload Search NF, TTL Flags, minor improvements and bug fixes.
```

Comment:
Major **Architectural**

Reply:
5/19 just means that release if up to May (we're just a tiny bit late 😄 )